src/appleseed/renderer/modeling/bsdf/glassbsdf.cpp

```diff
@@ -834,7 +904,7 @@ namespace
             const float denom = abs(4.0f * wo.y * wi.y);
             if (denom == 0.0f)
             {
-                value.set(0.0f);
+                value = SpectrumType(0.0f);
```

Comment:
That's expensive. Why the change?

Reply:
evaluate_reflection and evaluate_refraction have to work with both Spectrums and floats.
I'll push something better.